integreat_cms/cms/templates/pages/page_revisions.html

```diff
@@ -33,6 +33,10 @@ <h1 class="heading">
                 <b>{% translate "Author" %}:</b> <span id="revision-editor"></span>
                 <br />
                 <b>{% translate "Date" %}:</b> <span id="revision-date"></span>
+                {% if page_translation.first.hix_enabled and not page_translation.first.hix_ignore %}
```

Comment:
```suggestion
                {% if page_translations.first.hix_enabled and not page_translations.first.hix_ignore %}
```
Sorry, I made a small typo in my suggestion